### PR TITLE
Add local-gem workflow to CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by rogeruiz**
I've updated the Contributing docs to describe how to run the gem locally for website development. Please let me know what you think. Thanks!

CC: @maya @msecret @juliaelman

### CODE REVIEW

**Comment by juliaelman** [comment]
LGTM! :+1:

**Comment by juliaelman** [comment]
@maya @msecret would love to know your thoughts before merging in, per Slack discussion.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for the Jekyll Documentation Website
```
Comment (msecret) [inline CONTRIBUTING.md]: I think simplifying might help, something like "Local Development with Jekyll Documentation Website". I also question whether "Documentation Website" is the right term. Would contributors know what this is? Maybe we could show the link like "Documentation Website (playbook.cio.gov/designstandards)

**Comment by msecret** [comment]
Added a note about the language, but looking good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (maya) [inline CONTRIBUTING.md]: Agree, "Jekyll Documentation Website" is pretty wordy. How about "styleguide website"? Also, agree that we maybe don't need Gem in the title. Also we use sentence case on titles, so only the first word should be capitalized.

Looping in our content extraordinaire, @kategarklavs.

**Comment by rogeruiz** [comment]
Thanks @msecret and @maya. I updated the heading. Let me know if there is anything else we'd like to update here. I'll also wait for @kategarklavs to weigh in as well.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for [Styleguide Website](https://playbook.cio.gov/designstandards)
```
Comment (maya) [inline CONTRIBUTING.md]: Can this be sentence case?

Comment (maya) [inline CONTRIBUTING.md]: Also @kategarklavs, any thoughts on how we should refer to the website (e.g. styleguide website, Standards website, etc..)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
...
+
+If you'd like to see the changes to the Draft Web Design Standards in the local
+version of the website, you will need to run the `local-gem` gulp task before
+the `website:serve` task. This flag will generate a local version of the Ruby
+gem that the Jekyll site will use instead of the version described in the
+`Gemfile`.
+
+```sh
+gulp clean-dist && \ # Cleans up `dist-gem` if it exists
+gulp local-gem website:serve
```
Comment (msecret) [inline CONTRIBUTING.md]: I'm using these instructions and I'm not seeing my updated code on the local website. Is there something else that might have to be done here?

File: CONTRIBUTING.md (hunk 2 above)
Comment (rogeruiz) [inline CONTRIBUTING.md]: I can update this to say `Playbook website` / `documentation website`. I'll wait for @kategarklavs to weigh in as well.

**Comment by maya** [comment]
@rogeruiz is this still what we will be recommending? You mentioned the other day about not using the gem. Thanks!

**Comment by maya** [comment]
Closing and :fire: as we no longer have a local gem.

**Pull request closed without merging**